### PR TITLE
Feature/navbar

### PR DESCRIPTION
**Pull request by Rinkichi75**
Les layout sont regroupe par component:

Layout :  modification pour ajoute le navbar;
Event: ajout du dossier correspondant a la page evenement avec son test.
LifeStudent: ajout du dossier correspondant à la page Vie etudiante avec son test.
Navbar ajout du fichier navbar 
MainNavigator: Ajout du navbar

Par contre on dirait que les index.ts pour le fichier navbar n'a pas été commit enfin n'a pas voulu jsp pourquoi

### CODE REVIEW

File: src/components/Button/Button.component.tsx [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import React, { FunctionComponent } from 'react';
+
+export const Button: FunctionComponent = (name, img) => {
```
Comment (arnaud9145) [inline src/components/Button/Button.component.tsx]: name and img are not used, remove them

File: src/components/Button/Button.component.tsx [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+  return <button>Connection</button>;
```
Comment (arnaud9145) [inline src/components/Button/Button.component.tsx]: Do not put literals in code. Use i18n for translation. Plus, Button component should be a generic button, pass it a title props

File: src/components/Button/Button.component.tsx [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+};
```
Comment (arnaud9145) [inline src/components/Button/Button.component.tsx:9]: Button.style.scss is empty, remove it

File: src/components/Layout/Layout.component.tsx [hunk 4]
```diff
@@ -1,10 +1,13 @@
 import React, { FunctionComponent } from 'react';
+import { Navbar } from '../Navbar';
 import './Layout.style.scss';
 
 export const Layout: FunctionComponent = ({ children }) => {
   return (
     <div className="layout">
-      <div className="header">header</div>
+      <div className="header">
```
Comment (arnaud9145) [inline src/components/Layout/Layout.component.tsx]: Put that div in the Navbar component

File: src/components/Layout/Layout.component.tsx [hunk 5]
```diff
@@ -1,10 +1,13 @@
...
+        <Navbar></Navbar>
```
Comment (arnaud9145) [inline src/components/Layout/Layout.component.tsx]: <Navbar></Navbar> => <Navbar/>

File: src/components/Navbar/Navbar.component.tsx [hunk 6]
```diff
@@ -0,0 +1,50 @@
+import React, { FunctionComponent } from 'react';
+import city from './../../assets/city-hall.png';
+import home from './../../assets/home.png';
+import logo from './../../assets/logo.png';
+import timetable from './../../assets/timetable.png';
+import './Navbar.style.scss';
+
+const SideBarData = [
```
Comment (arnaud9145) [inline src/components/Navbar/Navbar.component.tsx]: you never use that variable

File: src/components/Navbar/Navbar.component.tsx [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+export const Navbar: FunctionComponent = () => {
+  return (
+    <nav>
+      <img className="logo" src={logo} height="256" width="256"></img>
+      <li className="nav-text">
+        <a href="/">
+          <img src={home} height="64" width="64"></img>
+          <span>Accueil</span>
+        </a>
+      </li>
```
Comment (arnaud9145) [inline src/components/Navbar/Navbar.component.tsx]: you repeat 3 times that code, factorise it in a component with Title, image, path as props

File: src/components/Navbar/Navbar.component.tsx [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+    title: 'Vie Etudiante',
+    path: 'vie_etudiante',
+    cName: 'nav-text',
+  },
+];
+
+export const Navbar: FunctionComponent = () => {
+  return (
+    <nav>
+      <img className="logo" src={logo} height="256" width="256"></img>
```
Comment (arnaud9145) [inline src/components/Navbar/Navbar.component.tsx]: height="256" width="256" => put those values in a const that you define at the top of the file, just after file import : const LOGO_IMAGE_SIZE = 256 and then => height={LOGO_IMAGE_SIZE} width={LOGO_IMAGE_SIZE}

File: src/components/Navbar/Navbar.component.tsx [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+      <li className="nav-text">
+        <a href="/">
+          <img src={home} height="64" width="64"></img>
```
Comment (arnaud9145) [inline src/components/Navbar/Navbar.component.tsx]: same here, put in const

File: src/components/Navbar/Navbar.style.scss [hunk 10]
```diff
@@ -0,0 +1,20 @@
+.nav {
+}
+.nav-text {
+  display: flex;
+  height: 60px;
+  text-decoration: none;
+  color: white;
+  font-size: 25px;
+}
+li {
```
Comment (arnaud9145) [inline src/components/Navbar/Navbar.style.scss]: avoid using native tags like li, div, ... prefer to use classnames, here put the code in .nav-text

File: src/modules/life_student/pages/LifeStudentPage/LifeStudentPage.component.tsx [hunk 11]
```diff
@@ -0,0 +1,5 @@
+import React, { FunctionComponent } from 'react';
+
+export const LifeStudentPage: FunctionComponent = () => {
```
Comment (arnaud9145) [inline src/modules/life_student/pages/LifeStudentPage/LifeStudentPage.component.tsx]: LifeStudentPage => StudentLifePage

Comment (arnaud9145) [reply src/modules/life_student/pages/LifeStudentPage/LifeStudentPage.component.tsx]: do not forget to rename it everywhere, in test too

File: src/modules/navigation/pages/MainNavigator/MainNavigator.component.tsx [hunk 12]
```diff
@@ -12,6 +14,12 @@ export const MainNavigator: FunctionComponent = () => {
           <Route exact path={PATHS.ROOT_PAGE.ROUTE}>
             <HomePage />
           </Route>
+          <Route exact path="/evenement">
```
Comment (arnaud9145) [inline src/modules/navigation/pages/MainNavigator/MainNavigator.component.tsx]: use PATHS for paths, see the Home page example above

**Review by arnaud9145 — APPROVED** [review]

File: src/modules/studentLife/pages/StudentLifePage/StudentLife.test.tsx [hunk 13]
```diff
@@ -0,0 +1,10 @@
+import { render } from '@testing-library/react';
+import React from 'react';
+import { StudentLifePage } from './StudentLifePage.component';
+
+describe('[Component] StudentLifePage', () => {
+  it('renders correctly', () => {
+    const studentLifePage = render(<StudentLifePage />);
+    expect(studentLifePage).toMatchSnapshot();
```
Comment (arnaud9145) [inline src/modules/studentLife/pages/StudentLifePage/StudentLife.test.tsx:8]: you did not run your tests, you are missing a snapshot here :) run `yarn test` to run all tests

Comment (Rinkichi75) [reply src/modules/studentLife/pages/StudentLifePage/StudentLife.test.tsx:8]: image

wtf C:\Users\Rinkichi\Desktop\etuutt-front\src\index.tsx
   1:27  error  Delete `␍`  prettier/prettier
   2:34  error  Delete `␍`  prettier/prettier
   3:23  error  Delete `␍`  prettier/prettier
   4:62  error  Delete `␍`  prettier/prettier
   5:60  error  Delete `␍`  prettier/prettier
   6:1   error  Delete `␍`  prettier/prettier
   7:17  error  Delete `␍`  prettier/prettier
   8:21  error  Delete `␍`  prettier/prettier
   9:22  error  Delete `␍`  prettier/prettier
  10:23  error  Delete `␍`  prettier/prettier
  11:34  error  Delete `␍`  prettier/prettier
  12:3   error  Delete `␍`  prettier/prettier

C:\Users\Rinkichi\Desktop\etuutt-front\src\modules\internationalization\service\i18n.service.ts
   1:28  error  Delete `␍`  prettier/prettier
   2:65  error  Delete `␍`  prettier/prettier
   3:50  error  Delete `␍`  prettier/prettier
   4:52  error  Delete `␍`  prettier/prettier
   5:1   error  Delete `␍`  prettier/prettier
   6:51  error  Delete `␍`  prettier/prettier
   7:64  error  Delete `␍`  prettier/prettier
   8:30  error  Delete `␍`  prettier/prettier
   9:55  error  Delete `␍`  prettier/prettier
  10:1   error  Delete `␍`  prettier/prettier
  11:22  error  Delete `␍`  prettier/prettier
  12:8   error  Delete `␍`  prettier/prettier
  13:1   error  Delete `␍`  prettier/prettier
  14:56  error  Delete `␍`  prettier/prettier
  15:13  error  Delete `␍`  prettier/prettier
  16:21  error  Delete `␍`  prettier/prettier
  17:23  error  Delete `␍`  prettier/prettier
  18:4   error  Delete `␍`  prettier/prettier
  19:1   error  Delete `␍`  prettier/prettier
  20:21  error  Delete `␍`  prettier/prettier

C:\Users\Rinkichi\Desktop\etuutt-front\src\modules\navigation\pages\index.ts
  1:33  error  Delete `␍`  prettier/prettier

C:\Users\Rinkichi\Desktop\etuutt-front\src\modules\navigation\services\index.ts
  1:35  error  Delete `␍`  prettier/prettier

C:\Users\Rinkichi\Desktop\etuutt-front\src\modules\navigation\services\routes.services.test.ts
   1:60  error  Delete `␍`  prettier/prettier
   2:1   error  Delete `␍`  prettier/prettier
   3:45  error  Delete `␍`  prettier/prettier
   4:47  error  Delete `␍`  prettier/prettier
   5:27  error  Delete `␍`  prettier/prettier
   6:54  error  Delete `␍`  prettier/prettier
   7:1   error  Delete `␍`  prettier/prettier
   8:32  error  Delete `␍`  prettier/prettier
   9:6   error  Delete `␍`  prettier/prettier
  10:1   error  Delete `␍`  prettier/prettier
  11:53  error  Delete `␍`  prettier/prettier
  12:31  error  Delete `␍`  prettier/prettier
  13:63  error  Delete `␍`  prettier/prettier
  14:1   error  Delete `␍`  prettier/prettier
  15:36  error  Delete `␍`  prettier/prettier
  16:6   error  Delete `␍`  prettier/prettier
  17:1   error  Delete `␍`  prettier/prettier
  18:54  error  Delete `␍`  prettier/prettier
  19:44  error  Delete `␍`  prettier/prettier
  20:76  error  Delete `␍`  prettier/prettier
  21:1   error  Delete `␍`  prettier/prettier
  22:43  error  Delete `␍`  prettier/prettier
  23:6   error  Delete `␍`  prettier/prettier
  24:4   error  Delete `␍`  prettier/prettier

C:\Users\Rinkichi\Desktop\etuutt-front\src\react-app-env.d.ts
  1:40  error  Delete `␍`  prettier/prettier

C:\Users\Rinkichi\Desktop\etuutt-front\src\setupTests.ts
  1:66  error  Delete `␍`  prettier/prettier
  2:33  error  Delete `␍`  prettier/prettier
  3:47  error  Delete `␍`  prettier/prettier
  4:59  error  Delete `␍`  prettier/prettier
  5:50  error  Delete `␍`  prettier/prettier

C:\Users\Rinkichi\Desktop\etuutt-front\src\translations\locale\index.ts
  1:28  error  Delete `␍`  prettier/prettier
  2:28  error  Delete `␍`  prettier/prettier
  3:1   error  Delete `␍`  prettier/prettier
  4:27  error  Delete `␍`  prettier/prettier

C:\Users\Rinkichi\Desktop\etuutt-front\src\__tests__\App.test.tsx
  1:49  error  Delete `␍`  prettier/prettier
  2:27  error  Delete `␍`  prettier/prettier
  3:30  error  Delete `␍`  prettier/prettier
  4:1   error  Delete `␍`  prettier/prettier
  5:36  error  Delete `␍`  prettier/prettier
  6:31  error  Delete `␍`  prettier/prettier
  7:33  error  Delete `␍`  prettier/prettier
  8:4   error  Delete `␍`  prettier/prettier

✖ 76 problems (76 errors, 0 warnings)
  76 errors and 0 warnings potentially fixable with the `--fix` option.

**Review by arnaud9145 — APPROVED** [review]

**Pull request merged**